### DISCUSSION TITLE
EFR32MG21 or CC2652P7 (SLZB-06M / SLZB-06p7)

### DISCUSSION THREAD

**Discussion by maciekdnd** [Koenkk/zigbee2mqtt#23145] [Devices]
Hello,

at this point of time and current development of ember firmware, is it better to get EFR32MG21 or CC2652P7 (SLZB-06M / SLZB-06p7)? Can someone ELI5, max devices connected directly (I found mixed info), speed, and stability with Z2M. Is it still experimental status with ember for EFR32? There is a lot of discussion but without conclusion that could be used to support purchase. I don't have experience with CC2652 chip, just EFR32.

Right now I have Sonoff Dongle-E, I will be switching to one of SLZB's for 60+ devices network with two buildings. 
Is CC2652P7 more stable than EFR32 with 7.4.3 and latest Z2M?
I have few Aqara switches (H1, D1, SSM-U01) and many motion sensors ZY-M100) that very often going offline and need to be repaired. Thank you for any helpful info.

**Comment by DubhAd** [comment]
I'd look at the MGM24 based sticks, there's been good feedback on those being stable on networks of 300+ devices.

You also should be able to migrate from your E to one of those, which you couldn't do with the CC2652 based stick.

**Reply by maciekdnd** [reply]
Thanks. MG24 is unavailable in my country at this moment, and I have to reconfigure my network in 2 days. I already decided to get SLZB-06M anyway. Ember is getting better and I will not get more than 150 devices. Only thing that would be nice to have better handling those pesky aqara/xiaomi/tuya devices. For that network I'm stuck at 7.4.1 + 1.3.1, so this may be the issue too.
It would be nice to have accurate and up-to-date information about all of that, as the current information is scattered across many different places.

**Reply by 1andrevich** [reply]
Knowing that ZY-M100 are often have problems like zigbee spam, you sure that your problem is coordinator? EFR32MG21 is quite capable chip

**Reply by maciekdnd** [reply]
Yeah, this is ancient info, already solved by new updates and fw. It is stable as a rock. I have 3 SLZB-06M, most of my sensors are replaced by better versions. I got rid of most of my ZY-M100's as those bastards are crazy (but not fully responsible of my previous network problems). AFAIK problem is in the chip firmware, even sensor manufacturer is unable to fix it (at least in this model).
